msteams-platform/toolkit/configure-bot-capability.md

```diff
@@ -0,0 +1,522 @@
+---
+title: Configure bot capability
+author: surbhigupta
+description: Learn how to configure bot capability within your Teams app.
+ms.author: v-preethah
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 02/16/2024
+---
+
+# Configure bot capability within your Teams app
+
+A bot, chatbot, or conversational bot is an app that responds to simple commands sent in chat and replies in meaningful ways. Bots that notify about build failures, provide information about the weather, bus schedules, or travel information are examples of bots in everyday use. A bot interaction can be a quick question and answer, or it can be a complex conversation. A bot can provide valuable and secure access to cloud services and corporate resources as a cloud application. For more information, see [build bots for Microsoft Teams](../bots/what-are-bots.md).
+
+## Prerequisites
+
+To configure bot as an additional capability, ensure the following prerequisites are met:
+
+* The app manifest (previously called Teams app manifest) file.
+* A [Microsoft 365 account](../concepts/build-and-test/prepare-your-o365-tenant.md) to test the application.
+* [Microsoft Azure Storage account](/azure/storage/common/storage-account-create).
+
+## Add bot to Teams app
+
+# [Add bot to Teams tab app](#tab/tabapp)
+
+The following steps help you to add bot to a tab app:
+
+* [Create bot Teams app using Microsoft Teams Toolkit](#create-bot-app-using-microsoft-teams-toolkit).
```

Comment:
When we are adding bot to the app how can we Create bot app?

It should be Create tab app using Microsoft Teams Toolkit